src/components/ShoppingList/styles.module.css

```diff
@@ -0,0 +1,40 @@
+.listElements {
+    width: 100%;
+}
+
+.newItem {
+    background-color: var(--item_grey);
+    color: var(--font_grey);
+    height: 40px;
+    border-radius: 20px;
+    margin-top: 10px;
+    box-sizing: border-box;
+    padding: 0 10px 0 28px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.newItem input,
+.newItem button[type=submit] {
+    background-color: transparent;
+    outline: none;
+    border: none;
+}
+
+.newItem input {
+    border-bottom: 2px solid var(--main_green);
+    font-size: 12px;
+    font-weight: 700;
```

Comment:
Does this need a `flex-grow: 1;` so it doesn't start scrolling for longer items?
![long names mov](https://user-images.githubusercontent.com/9036408/97082515-3f23f980-1602-11eb-9914-f2fc7d1aa233.gif)

Reply:
Most likely! I never actually typed in a longer name to test it 🤦‍♂️